### $\text{Assignment II}$

$\text{State Space of Matrix S = {0,1,2}}$

$\text{Given Probabilities:}$
+ ${P_{00} = 0.7, P_{01} = 0.2}$
+ ${P_{10} = 0.4, P_{12} = 0.0}$
+ ${P_{20} = 0.0, P_{21} = 1.0, P_{22} = 0.0}$

#### $\text{(a)}$
$\text{Using the law of total probability, we can calculate the missing probabilities as follows:}$

$P_{02} = 1 - P_{00} - P_{01} = 0.1$

$P_{11} = 1 - P_{10} - P_{12} = 0.6$

$\text{So we obtain the following transition matrix:}$

$\text{[0.7, 0.2, 0.1]}$

$\text{[0.4, 0.6, 0.0]}$

$\text{[0.0, 1.0, 0.0]}$

#### $\text{(b)}$

$\text{Given that the Markov Chain was initially in State 1, initial state vector}$
${\pi_{0} =}\text{ [0,1,0] }$

$\text{Using Kolmogorov's equation }{P^{m+n}_{ij} = \sum_{k \in S} P^{m}_{ik}P^{n}_{kj}\text{ }\forall\text{ }i,j\text{ }\in\text{ }S,}$
$\text{we see that }{ \pi_{n} = \pi_{0} P^{n}}$

$\text{We calculate } \pi_{n} \text{ for n = 10,20,50 using the same formula below:}$

In [34]:
# initializing transition matrix
C1 <- c(0.7, 0.4, 0.0)
C2 <- c(0.2, 0.6, 1.0)
C3 <- c(0.1, 0.0, 0.0)
P <- matrix(c(C1,C2,C3),nrow=3,ncol=3)
print(P)

# initializing state vector
p <- matrix(c(0,1,0),nrow=1,ncol=3)
print(p)

     [,1] [,2] [,3]
[1,]  0.7  0.2  0.1
[2,]  0.4  0.6  0.0
[3,]  0.0  1.0  0.0
     [,1] [,2] [,3]
[1,]    0    1    0


In [35]:
# calculate P 10, P 20, P 50
transition = diag(3)

for(i in 1:50)
{
    p <- p %*% P
    transition <- transition %*% P
    
    if(i==10)
    {
        print("-------------------------------")
        print("P 10")
        print(transition)        
        print("pi 10")
        print(p)
    }
    
    if(i==20)
    {
        print("-------------------------------")
        print("P 20")
        print(transition)  
        print("pi 20")
        print(p)
    }
    
    if(i==49)
    {
        print("-------------------------------")
        print("P 50")
        print(transition)
        print("pi 50")
        print(p)
        print("-------------------------------")
    }
}

[1] "-------------------------------"
[1] "P 10"
          [,1]      [,2]       [,3]
[1,] 0.5405406 0.4054053 0.05405408
[2,] 0.5405405 0.4054055 0.05405402
[3,] 0.5405402 0.4054058 0.05405399
[1] "pi 10"
          [,1]      [,2]       [,3]
[1,] 0.5405405 0.4054055 0.05405402
[1] "-------------------------------"
[1] "P 20"
          [,1]      [,2]       [,3]
[1,] 0.5405405 0.4054054 0.05405405
[2,] 0.5405405 0.4054054 0.05405405
[3,] 0.5405405 0.4054054 0.05405405
[1] "pi 20"
          [,1]      [,2]       [,3]
[1,] 0.5405405 0.4054054 0.05405405
[1] "-------------------------------"
[1] "P 50"
          [,1]      [,2]       [,3]
[1,] 0.5405405 0.4054054 0.05405405
[2,] 0.5405405 0.4054054 0.05405405
[3,] 0.5405405 0.4054054 0.05405405
[1] "pi 50"
          [,1]      [,2]       [,3]
[1,] 0.5405405 0.4054054 0.05405405
[1] "-------------------------------"


$\text{We observe that the Markov Chain attains a limiting distribution at }{P^{20}}\text{ which is}$
$\text{expected since the chain is irreducible as well as aperiodic.}$
$\text{This is because the state space itself represents a recurrent communicating class}$
$\text{and all the states are aperiodic.}$

$\text{Since the existence of a limiting distribution implies a steady state, we obtain the }$
$\text{stationary distribution vector }$
${[\pi_{0},\pi_{1},\pi_{2}] = [0.540, 0.405, 0.054]}$

#### $\text{Ashwin Singh}$

#### $\text{2017222}$